tests/framework/cli/test_cli.py

```diff
@@ -329,14 +329,14 @@ def test_init_error_is_caught(self, entry_points, entry_point):
 
 class TestKedroCLI:
     def test_project_commands_no_clipy(self, mocker, fake_metadata):
-        mocker.patch(
-            "kedro.framework.cli.cli.importlib.import_module",
-            side_effect=cycle([ModuleNotFoundError()]),
-        )
         mocker.patch("kedro.framework.cli.cli._is_project", return_value=True)
         mocker.patch(
             "kedro.framework.cli.cli.bootstrap_project", return_value=fake_metadata
         )
+        mocker.patch(
+            "kedro.framework.cli.cli.importlib.import_module",
+            side_effect=cycle([ModuleNotFoundError()]),
+        )
```

Comment:
`import_module` need to be the last mock, otherwise it makes all subsequent `mocker.patch` fails